ci/deploy_website/website.sh

```diff
@@ -28,8 +28,7 @@ set -euo pipefail
 # or change how we want bugs to be filed.
 declare -A shortlinks=(
     [bug]="https://github.com/MaterializeInc/materialize/issues/new?labels=C-bug&template=01-bug.yml"
-    [non-materialized-error]="https://materialize.com/docs/sql/create-view/#querying-non-materialized-views"
```

Comment:
Oh, good catch that this broke, but the hope was to never remove one of these links! Can you re-point at the LTS docs? https://materialize.com/docs/lts/sql/create-view/#querying-non-materialized-views